src/content/docs/crowdin/translation-process/pre-translation.mdx

```diff
@@ -68,6 +68,20 @@ To apply Pre-translation manually, follow these steps:
   If the [TM Auto-substitution](/translation-memory/#tm-auto-substitution) feature is enabled when pre-translating via TM, it means that TM suggestions improved to 100% will be treated as regular 100% matches.
 </Aside>
 
+### Pre-translation Queue
+
+The Pre-translation queue allows you to trigger multiple manual pre-translations via TM, MT, or AI, each with different settings. After initiating a pre-translation, each instance enters a queue with a status of **Pending**.
+
+Upon triggering a manual pre-translation, you will receive a notification indicating that the pre-translation has been added to the queue. To review all pre-translations currently in the queue, navigate to **Tools > Pre-translation queue** in your project. Alternatively, click **Pre-translation** in the project's **Dashboard** and select **Queue** to be redirected directly to the **Tools > Pre-translation queue**.
+
+Each entry in the Pre-translation queue table includes the following details:
+
+- **Created at**
```

Comment:
This option with no description looks strange, please consider adding some description

Reply:
Added a description